### ISSUE TITLE
File is wiped if when parsing invalid player_config.json

### ISSUE THREAD

**Issue by corndogit** [seraxis/lr2oraja-endlessdream#38]
**Description**

When the player's config is read and does not validate successfully, the file is wiped and replaced with a completely new config. This bug has happened to me firsthand as well as others, and I believe is a known issue although hasn't been raised as an ED issue.

**Steps to reproduce:**

- Open player_config.json in a text editor and edit it to be invalid JSON
- Launch the beatoraja launcher

**Result**: config is reset to default settings

**Root cause:** code failure in `bms.player.beatoraja.PlayerConfig` (around Line 888)

```java
if(readAllPlayerID(config.getPlayerpath()).length == 0 || readPlayerConfig(config.getPlayerpath(), config.getPlayername()) == null) {
    PlayerConfig pc = new PlayerConfig();
    ...
}
```

Looking at this condition in PlayerConfig.java, it seems the `init` method makes a `readPlayerConfig` call to read the config. If an exception is thrown for *any* reason, all that happens is that the stack trace gets printed, so this method returns null. The code inside this condition block then creates a new config file, overwriting the one with the same name.

**Comment by corndogit** [comment]
Will work on a fix for it this week if I have time

**Comment by corndogit** [comment]
Another bug exists in PlayerConfig.java:979

`Files.copy(path, Paths.get(playerpath + "/" + playerid + "/config_backup.json"));`

By default, the copy fails if the target file already exists or is a symbolic link, unless the REPLACE_EXISTING copy option is specified. So if a backup file already exists, the backup won't be written

**Comment by seraxis** [comment]
fixed in #39
